src/tools/miri/src/shims/windows/foreign_items.rs

```diff
@@ -267,6 +267,14 @@ pub trait EvalContextExt<'mir, 'tcx: 'mir>: crate::MiriInterpCxExt<'mir, 'tcx> {
 
                 this.Sleep(timeout)?;
             }
+            "CreateWaitableTimerExW" => {
+                let [_attributes, _name, _flags, _access] =
+                    this.check_shim(abi, Abi::System { unwind: false }, link_name, args)?;
```

Comment:
Generally we do at least read all the arguments of a shim to make sure they are initialized -- or we mark the shim as "only run in std".

Reply:
Ok, I've added the reads.

Reply:
Thanks! LGTM.